### PR TITLE
Update wine prefrences input

### PR DESCRIPTION
**Pull request by plapnt87**
Add label to Preferences Input
Add Save button
Improve style wine info
Add other Preferences options

### CODE REVIEW

**Comment by plapnt87** [comment]
@Mattia46 I have made a PR for the wine preferences page. Have a look and let me know.
I didn't manage to fix the issue with the numpad. Looks like need some js magic. I will try to look more on it.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 1]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            />
+            {/* Body */}
+            <div className={style.c_subtitle__input}>Body</div>
+            <Dropdown
+              dropdownName="alcohol"
+              options={["Light", "Medium", "Full"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
+            />
+            {/* Sweetness */}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Serve?

File: src/Pages/WinePreferences/WinePreferences.js [hunk 2]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            <div className={style.c_subtitle__input}>Sweetness</div>
+            <Dropdown
+              dropdownName="alcohol"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: sweetness invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 3]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              wineChange={wineChange}
+              selected={wine.alcohol}
+            />
+            {/* Tannin */}
+            <div className={style.c_subtitle__input}>Tannin</div>
+            <Dropdown
+              dropdownName="alcohol"
+              options={["Low", "Medium", "High"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: sweetness invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 4]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            />
+            {/* Complexity */}
+            <div className={style.c_subtitle__input}>Complexity</div>
+            <Dropdown
+              dropdownName="alcohol"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: complexity invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 5]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Low", "Medium", "High"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: complexity invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 6]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            />
+            {/* Tannin */}
+            <div className={style.c_subtitle__input}>Tannin</div>
+            <Dropdown
+              dropdownName="alcohol"
+              options={["Low", "Medium", "High"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
+            />
+            {/* Complexity */}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Serve?

File: src/Pages/WinePreferences/WinePreferences.js [hunk 7]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            <div className={style.c_subtitle__input}>Complexity</div>
+            <Dropdown
+              dropdownName="alcohol"
+              options={["Low", "Medium", "High"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
+            />
+            {/* Balance */}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Serve?

File: src/Pages/WinePreferences/WinePreferences.js [hunk 8]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            <div className={style.c_subtitle__input}>Balance</div>
+            <Dropdown
+              dropdownName="alcohol"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Balance invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 9]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Low", "Medium", "High"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Balance invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 10]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            />
+            {/* Maturity */}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: ?

File: src/Pages/WinePreferences/WinePreferences.js [hunk 11]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            <div className={style.c_subtitle__input}>Maturity</div>
+            <Dropdown
+              dropdownName="alcohol"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Maturity invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 12]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={[
+                "Immature",
+                "Ready to drink/can keep",
+                "Ready to drink",
+                "Pass his prime"
+              ]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Maturity invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 13]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            />
+            {/* Finish */}
+            <div className={style.c_subtitle__input}>Finish</div>
+            <Dropdown
+              dropdownName="alcohol"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Finish invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 14]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Short", "Medium", "Long"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Finish invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 15]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={[
+                "Immature",
+                "Ready to drink/can keep",
+                "Ready to drink",
+                "Pass his prime"
+              ]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
+            />
+            {/* Finish */}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: ?

File: src/Pages/WinePreferences/WinePreferences.js [hunk 16]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            <div className={style.c_subtitle__input}>Finish</div>
+            <Dropdown
+              dropdownName="alcohol"
+              options={["Short", "Medium", "Long"]}
+              wineChange={wineChange}
+              selected={wine.alcohol}
+            />
+            {/* Listing candidate */}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: ?

File: src/Pages/WinePreferences/WinePreferences.js [hunk 17]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+            <div className={style.c_subtitle__input}>Listing candidate</div>
+            <Dropdown
+              dropdownName="alcohol"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: listingCandidate invece di alcohol

File: src/Pages/WinePreferences/WinePreferences.js [hunk 18]
```diff
@@ -28,45 +28,122 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Regular", "By the glass"]}
               wineChange={wineChange}
               selected={wine.alcohol}
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: listingCandidate invece di alcohol

File: src/Pages/WinePreferences/Dropdown/Dropdown.js [hunk 19]
```diff
@@ -19,13 +19,13 @@ const Dropdown = ({
                 value={selected}
                 className={style.c_dropdown}
             >
-                <option> -- please select on --</option>
+                <option>Select</option>
```
Comment (Mattia46) [inline src/Pages/WinePreferences/Dropdown/Dropdown.js]: Qst dovrebbe essere cosi:
```
<option> --- please select {dropdownName} -- </option>
```

File: src/Pages/WinePreferences/Dropdown/Dropdown.js [hunk 20]
```diff
@@ -19,13 +19,13 @@ const Dropdown = ({
...
                 {optionList()}
             </select>
-
+{/*
```
[the remark this answers is not in the document]
Comment (Mattia46) [reply src/Pages/WinePreferences/Dropdown/Dropdown.js]: Can you remove the comments?

File: src/Pages/WinePreferences/WinePreferences.js [hunk 21]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+
           <div onClick={() => setShowWineNote(true)} className={style.btNotes}>
             {wine.note}
           </div>
           <div className={style.c_subtitle}>Preferences</div>
           <div style={{ display: "flex", flexDirection: "column" }}>
+
+            <div className={style.c_subtitle__input}>Intensity</div>
+            <Dropdown
+              dropdownName="Intensity"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: The name has to be lowerCase like `{wine.intensity}`

File: src/Pages/WinePreferences/WinePreferences.js [hunk 22]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Low", "Medium", "Pronounced"]}
+              wineChange={wineChange}
+              selected={wine.intensity}
+            />
+
+            <div className={style.c_subtitle__input}>Flavour</div>
             <Dropdown
-              dropdownName="acidity"
-              options={["low", "medium", "high"]}
+              dropdownName="Flavour"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 23]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Fruit", "Floral", "Spice", "Vegetal", "Nut", "Oak"]}
+              wineChange={wineChange}
+              selected={wine.flavour}
+            />
+
+            <div className={style.c_subtitle__input}>Acidity</div>
+            <Dropdown
+              dropdownName="Acidity"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 24]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Low", "Medium", "High"]}
               wineChange={wineChange}
               selected={wine.acidity}
             />
+
+            <div className={style.c_subtitle__input}>Body</div>
+            <Dropdown
+              dropdownName="Body"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 25]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Light", "Medium", "Full"]}
+              wineChange={wineChange}
+              selected={wine.body}
+            />
+
+            <div className={style.c_subtitle__input}>Sweetness</div>
+            <Dropdown
+              dropdownName="Sweetness"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 26]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Dry", "Off dry", "Sweet"]}
+              wineChange={wineChange}
+              selected={wine.sweetness}
+            />
+
+            <div className={style.c_subtitle__input}>Tannin</div>
             <Dropdown
-              dropdownName="alcohol"
-              options={["low", "medium", "high"]}
+              dropdownName="Tannin"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 27]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Low", "Medium", "High"]}
               wineChange={wineChange}
-              selected={wine.alcohol}
+              selected={wine.tannin}
+            />
+
+            <div className={style.c_subtitle__input}>Complexity</div>
+            <Dropdown
+              dropdownName="Complexity"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 28]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Low", "Medium", "High"]}
+              wineChange={wineChange}
+              selected={wine.complexity}
+            />
+
+            <div className={style.c_subtitle__input}>Balance</div>
+            <Dropdown
+              dropdownName="Balance"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 29]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Low", "Medium", "High"]}
+              wineChange={wineChange}
+              selected={wine.balance}
+            />
+
+            <div className={style.c_subtitle__input}>Maturity</div>
+            <Dropdown
+              dropdownName="Maturity"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 30]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+                "Ready to drink",
+                "Pass his prime"
+              ]}
+              wineChange={wineChange}
+              selected={wine.maturity}
+            />
+
+            <div className={style.c_subtitle__input}>Finish</div>
+            <Dropdown
+              dropdownName="Finish"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: Same, lowercase.

File: src/Pages/WinePreferences/WinePreferences.js [hunk 31]
```diff
@@ -28,47 +28,124 @@ const WinePreferences = ({ history, updateWine }) => {
...
+              options={["Short", "Medium", "Long"]}
+              wineChange={wineChange}
+              selected={wine.finish}
+            />
+
+            <div className={style.c_subtitle__input}>Listing candidate</div>
+            <Dropdown
+              dropdownName="Listing candidate"
```
Comment (Mattia46) [inline src/Pages/WinePreferences/WinePreferences.js]: `listingCandidate`

**Pull request merged**